debug_windows.bat

```diff
@@ -0,0 +1,59 @@
+@echo off
+
+cd /D "%~dp0"
+
+echo Detecting Zig compiler...
+
+set /p baseVersion=<".zig-version"
+
+IF "%PROCESSOR_ARCHITECTURE%"=="AMD64"	(set arch=x86_64)
+IF "%PROCESSOR_ARCHITECTURE%"=="IA64"	(set arch=x86_64)
+IF "%PROCESSOR_ARCHITECTURE%"=="x64"	(set arch=x86)
```

Comment:
x64 is also x86_64
`%PROCESSOR_ARCHITECTURE%` seems to be `x86` on 32 bit platforms.

Reply:
Typo. `x64` does not actually exist.